amplipi/updater/asgi.py

```diff
@@ -59,10 +62,42 @@ class ReleaseInfo(BaseModel):
 dir_path = os.path.dirname(real_path)
 app.mount("/static", StaticFiles(directory=f"{dir_path}/static"), name="static")
 
-HOME = f"{os.environ.get('HOME')}/amplipi-dev" # standard install directory
+INSTALL_DIR = os.getenv('INSTALL_DIR', os.getcwd())
+
+# if we have a broken configuration, the updater should still function
+# as a failsafe
+try:
+  with open(f"{INSTALL_DIR}/house.json") as f:
+    config = json.load(f)
+except:
+  config = {"info": {}}
```

Comment:
a little brittle considering future other uses of the config in the updater , but oughta function for now. is that copacetic?